src/main/kotlin/it/pagopa/aca/config/WebClientConfig.kt

```diff
@@ -47,8 +52,16 @@ class WebClientConfig {
                 connectionTimeout = connectionTimeout,
                 it.pagopa.generated.gpd.ApiClient.buildWebClientBuilder()
             )
+        val dateFormat = SimpleDateFormat(GPD_DATE_FORMAT)
```

Comment:
I removed the format set in the client because with the generation of the code with the LocalDateTime we implicitly solved the problem 🙂 fixed with 👉🏻 0375c055a87eac221aae9039b4a35277ba3b22a8